api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -1781,6 +1781,9 @@ message ListUnifiedResourcesRequest {
   // UsePreviewAsRoles indicates that the response should include all resources
   // the caller would be able to access with their preview_as_roles
   bool UsePreviewAsRoles = 10 [(gogoproto.jsontag) = "use_preview_as_roles,omitempty"];
+  // PinnedResources indicates that the request will pull only the pinned resources
+  // of the requesting user
+  bool PinnedResources = 11 [(gogoproto.jsontag) = "pinned_resources,omitempty"];
```

Comment:
Weird that the linter says this field isn't present

Reply:
How about PinnedResourcesOnly for a name?

Reply:
Thank you! I was trying to think of a name like "include pinned resources" or "use pinned resources" but every name sounded inclusive instead of exclusive so I gave up. I like Only. Will update.